pyiron/atomistics/structure/atoms.py

```diff
@@ -2598,6 +2598,14 @@ def get_distances_array(self, a0=None, a1=None, mic=True, vector=False):
         else:
             return dist
 
+    def append(self, atom):
+        if isinstance(atom, Atom) or isinstance(atom, ASEAtom):
```

Comment:
Since our `Atom` is derived from `ASEAtom`, `isinstance(atom, ASEAtom)` should be enough